packages/frontend/app/components/LiquidityDialog.tsx

```diff
@@ -41,10 +66,21 @@ export const LiquidityDialog = ({
                 <Form method='post' replace preventScrollReset>
```

Comment:
```suggestion
                <Input
                required
                type='number'
                name='displayAmount'
                label='Amount'
                onChange={handleChange}
                addOn={asset.code}
                step='any'
                min={1 / Math.pow(10, asset.scale)}
                error={errorMessage}
                />
                <Form method='post' replace preventScrollReset>
                  <Input
                    required
                    min={1}
                    type='hidden'
                    name='amount'
                    value={actualAmount}
                  />
                  <div className='flex justify-end py-3'>
                    <Button aria-label={`${type} liquidity`} type='submit'>
                      {type} liquidity
                    </Button>
                  </div>
                </Form>
```

This is a really small (and last) suggestion because everything looks great! But if we move this dummy input field above the form it actually doesn't get sent to the backend when the form is submitted. This way we keep our data collection cleaner on the backend and make sure no-one working on this in the future gets confused about which amount to use for making the GraphQL request or anything like that, we only send what we actually expect to receive on the server-side.